### PR TITLE
Favorites View : X Button added to Unlike Items

### PR DESCRIPTION
**Pull request by cbaja**
X Button added to unlike items. 
The action is not yet implemented.

@scottrichards @smacgregor

### CODE REVIEW

File: app/src/main/res/layout/pet_favorites_items.xml [hunk 1]
```diff
@@ -34,6 +34,16 @@
...
     <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentRight="true"
+        android:id="@+id/tvRemovePetFav"
+        android:layout_marginTop="5dp"
+        android:textColor="#363636"
+        android:drawableLeft="@drawable/ic_remove_petfav"
+        android:gravity="center_vertical"/>
+
```
Comment (smacgregor) [inline app/src/main/res/layout/pet_favorites_items.xml:49]: maybe you did this in one of the other pull requests but I think we'd want the unlike button to be an actual button and not  background image on a text view. Checkout the ImageButton element which you can specify in an xml file. 

That will make it easier to have press states on the button as well.

**Comment by smacgregor** [comment]
Carly, what do you think about showing our existing heart icon in the row as the image button icon instead of an X button? It could be a heart with a red tint so it looks red. Tapping the heart would have a push state that turns it back to grey and then the row is removed.

As a user it might be hard to understand what the X means for a favorite pet. The heart might re-enforce what's happening a little better.

**Comment by scottrichards** [comment]
I really like Scott's suggestion that is a good idea.

**Comment by scottrichards** [comment]
I think the heart should be bigger and it is pretty narrow.  I would make it consistent with the heart in the action bar and the Card view.

**Pull request merged**